step-templates/sql-deploy-dacpac.json

```diff
@@ -3,10 +3,10 @@
}'.\" -f $PackageStepName, $InstallPath)\n            return $InstallPath\n        }\n\n        throw \"Could not determine the install location of the package.\"\n    }\n    catch\n    {\n        $_ | Format-List -Force | Out-String | Write-Debug;\n        throw;\n    }\n}\n\n# Get the supplied parameters\n$DACPACPackageStep = $OctopusParameters[\"DACPACPackageStep\"]\n$DACPACPackageName = $OctopusParameters[\"DACPACPackageName\"]\n$PublishProfile = $OctopusParameters[\"DACPACPublishProfile\"]\n$Report = Format-OctopusArgument -Value $OctopusParameters[\"Report\"]\n$Script = Format-OctopusArgument -Value $OctopusParameters[\"Script\"]\n$Deploy = Format-OctopusArgument -Value $OctopusParameters[\"Deploy\"]\n$ExtractTargetDatabaseDacpac = Format-OctopusArgument -Value $OctopusParameters[\"ExtractTargetDatabaseDacPac\"]\n$TargetServer = $OctopusParameters[\"TargetServer\"]\n$TargetDatabase = $OctopusParameters[\"TargetDatabase\"]\n$TargetDatabaseVersion = $OctopusParameters[\"TargetDatabaseVersion\"]\n$UseIntegratedSecurity = Format-OctopusArgument -Value $OctopusParameters[\"UseIntegratedSecurity\"]\n$Username = $OctopusParameters[\"SQLUsername\"]\n$Password = $OctopusParameters[\"SQLPassword\"]\n$EnableMultiSubnetFailover = Format-OctopusArgument -Value $OctopusParameters[\"EnableMultiSubnetFailover\"]\n$AdditionalDeploymentContributors = Format-OctopusArgument -Value $OctopusParameters[\"AdditionalContributors\"]\n$AdditionalDeploymentContributorArguments = Format-OctopusArgument -Value $OctopusParameters[\"AdditionalContributorArguments\"]\n\n$InstallPath = Get-DacpacInstallPath -PackageStepName $DACPACPackageStep\nif(!(Test-Path $InstallPath)) {\n    Throw (\"The package extraction folder '{0}' does not exist or the Octopus tentacle does not have permission to access it.\" -f $InstallPath)\n}\n\n# Expand the publish dacpac filename\n$DACPACPackageName = ($InstallPath + \"\\\" + $DACPACPackageName)\nif(!(Test-Path $DACPACPackageName)) {\n    Throw (\"Could not find the file '{0}'\" -f $DACPACPackageName)\n}\n\n# Expand the publish profile filename if a value has been supplied\nIf ($PublishProfile)\n{\n    $PublishProfile = ($InstallPath + \"\\\" + $PublishProfile)\n    if(!(Test-Path $PublishProfile)) {\n        Throw (\"Could not find the file '{0}'\" -f $PublishProfile)\n    }\n}\n\n# Invoke the DacPac utility\ntry\n{\n    Invoke-DacPacUtility -Report $Report -Script $Script -Deploy $Deploy -ExtractTargetDatabaseDacpac $ExtractTargetDatabaseDacpac -DacPacFilename $DACPACPackageName -TargetServer $TargetServer -TargetDatabase $TargetDatabase -UseIntegratedSecurity $UseIntegratedSecurity -Username $Username -Password $Password -EnableMultiSubnetFailover $EnableMultiSubnetFailover -PublishProfile $PublishProfile -AdditionalDeploymentContributors $AdditionalDeploymentContributors -AdditionalDeploymentContributorArguments $AdditionalDeploymentContributorArguments\n}\ncatch\n{\n    $_.Exception.Message\n    $_.Exception.StackTrace \n    Exit 1\n}",
```

Comment:
```
$InstallPathCustomDirectoryKey = ("Octopus.Action[{0}].Package.CustomInstallationDirectory" -f $DACPACPackageStep)
$InstallPath = $OctopusParameters[$InstallPathCustomDirectoryKey]
if ($InsallPath -ne $null) {
    Write-Verbose ("Install path determined from '{0}' custom installation directory parameter as '{1}'." -f $PackageStepName, $InstallPath)
    return $InstallPath
}
```

Typo in parameter used in the `if` statement, `$insallPath` instead of `$installPath`